### PR TITLE
Bumping into grilles damages them

### LINKED ISSUE #3586: Ghetto powersink is too strong
You can construct a ghetto powersink by using a monkey on a conveyor belt that leads directly into an electrified grille. This causes power drain in excess of several MW (probably much higher as I did not actually reach a limit) and will easily prevent all networked APCs from receiving power.

I'm not sure what would be the best course of nerfing this aside from removing power drain from shocks.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Deals 0-1 damage every time a grille is bumbed. Will take an average of 100 bumps to fully destroy a grille.

## Why It's Good For The Game

Prevents using them to drain all power from a network and creating server lag by conveyoring mobs into them.

## Changelog
:cl:
tweak: Grilles now take damage when a mob bumps into them.
/:cl:

### CODE REVIEW

**Comment by eeSPee** [comment]
I'd go for 0 - 3, because it overheats and eventually melts down.

**Comment by Sinestia** [comment]
tarkov barb wire scratch sound
/s
oh wait i read it as damaging the user

that makes sense

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -84,6 +84,7 @@
 		return
 	var/mob/M = AM
 	shock(M, 70)
+	take_damage(rand(0,1), BRUTE, "melee", FALSE)
```
Comment (Archanial) [inline code/game/objects/structures/grille.dm]: what's the point of calling damage with 0?
```suggestion
	if(prob(50))
		take_damage(1, BRUTE, "melee", FALSE)
```

Comment (PowerfulBacon) [reply code/game/objects/structures/grille.dm]: 🤷

**Comment by Archanial** [comment]
obraz
no, the average would be 200

**Review by utilbird — APPROVED** [review]

**Pull request merged**